Taskfile.yml

```diff
@@ -99,10 +99,11 @@ tasks:
         go test \
           -v \
           -short \
+          {{ .GO_TEST_PACKAGE }} \
           -run '{{default ".*" .GO_TEST_REGEX}}' \
           {{default "-timeout 20m -coverpkg=./... -covermode=atomic" .GO_TEST_FLAGS}} \
           -coverprofile=coverage_unit.txt \
-          {{default .DEFAULT_GO_PACKAGES .GO_PACKAGES}} \
+          {{default .DEFAULT_INTEGRATIONTEST_GO_PACKAGES .GO_PACKAGES}} \
```

Comment:
Doing so removes the unit-tests from the test suite.

Before `DEFAULT_GO_PACKAGES` had both unit and integration tests, after the split `DEFAULT_GO_PACKAGES` now has the unit tests and `DEFAULT_INTEGRATIONTEST_GO_PACKAGES` has the integration tests. BTW this change is not reflected in the TaskFile targets, I'd expect to have a `go:test` target to run unit-tests and a `go:integration-test` to run the unit tests.

The same separation should be done in the github workflows, the integration-test should be run under the os+package matrix, while the unit test should be under the os matrix.